src/imgui/ImGuiReverseBar.cc

```diff
@@ -190,22 +190,52 @@ void ImGuiReverseBar::showMenu(MSXMotherBoard* motherBoard)
 				ImGui::TextUnformatted("No replays found"sv);
 			} else {
 				ImGui::TextUnformatted("Select replay"sv);
-				im::ListBox("##select-replay", [&]{
-					for (const auto& [fullName_, displayName_, _] : replayNames) {
+				int flags = ImGuiTableFlags_RowBg |
+					ImGuiTableFlags_BordersV |
+					ImGuiTableFlags_BordersOuter |
+					ImGuiTableFlags_Resizable |
+					ImGuiTableFlags_Sortable |
+					ImGuiTableFlags_Hideable |
+					ImGuiTableFlags_Reorderable |
+					ImGuiTableFlags_ContextMenuInBody |
+					ImGuiTableFlags_SizingStretchProp;
+				im::Table("##select-replay", 2, flags, {}, [&]{
+					for (const auto& [fullName_, displayName_, ftime_] : replayNames) {
 						const auto& fullName = fullName_; // clang workaround
-						const auto& displayName = displayName_; // clang workaround
-						if (ImGui::Selectable(displayName.c_str())) {
-							manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
-						}
-						im::PopupContextItem([&]{
+						if (ImGui::TableNextColumn()) {
+							const auto& displayName = displayName_; // clang workaround
+							if (ImGui::Selectable(displayName.c_str(), false, ImGuiSelectableFlags_SpanAllColumns | ImGuiSelectableFlags_AllowOverlap)) {
+								manager.executeDelayed(makeTclList("reverse", "loadreplay", fullName));
+							}						im::PopupContextItem([&]{
 							if (ImGui::MenuItem("delete")) {
 								confirmCmd = makeTclList("file", "delete", fullName);
 								confirmText = strCat("Delete replay '", displayName, "'?");
 								openConfirmPopup = true;
 							}
 						});
-					}
-				});
+
+						}
+						if (ImGui::TableNextColumn()) {
+							const auto& ftime = ftime_; // clang workaround
```

Comment:
Is this workaround needed? (Did it fail without this on clang?)
The "problem" occurs when an element of a structured-binding (the `const auto& [a, b, c]` stuff) is later used as a capture in a lambda function. I don't believe that is the case here.

Is it too soon to bump the required clang version to clang-16? Then we no longer need these workarounds.

Reply:
I have no idea what the status is on macOS now. So I can't tell. @grauw which clang version is safe to use on macOS, can you shed some light on this?

Reply:
The Clang version that is safe to use on macOS is Apple Clang version 15.0.0 (clang-1500.3.9.4) as shipped in Xcode 15.4.

Note that Apple Clang version numbers do not map to LLVM Clang, this is not the same as LLVM Clang 15. So if you’re asking which LLVM Clang version is safe to use, I can’t answer that question. If you don’t have access to macOS, just let the CI build it and see if it complains. That’s how I deal with MSVC and GCC build issues as well.

Reply:
Thanks. We should probably do some follow up commits to see whether the workaround is still needed on the latest macOS images.

Reply:
If you do a PR or test in a fork (or a branch when you add it to on: push: branches: in build_snapshots.yml) the CI will build without having to push it to master.

Reply:
@grauw we actually already have that. See below, this PR is built automatically. No need to change the yml file.

Reply:
Just outlining multiple ways to get the CI to trigger other than opening a PR, to test Apple Clang, for future reference :). Since PRs cause all kinds of notifications to people, and additional ones for any iterations on it, depending on what you’re trying to do one may not always want that before the patch is ready.